tools/test/examples/examples_lib.py

```diff
@@ -493,7 +493,17 @@ def symlink_mbedos(config, path, exp_filter):
     for example in config['examples']:
         if example['name'] not in exp_filter:
             continue
-        for name in get_sub_examples_list(example):
+
+        # it's not important that all sub examples are defined in exmaples.json to build
```

Comment:
```suggestion
        # it's not important that all sub examples are defined in examples.json to build
```

Reply:
I dont understand the comment, what is not important and how this is related what is below?

Reply:
I have updated the comment to make the comment in source code simple. The actual reason for change is mentioned in PR description.